### PR TITLE
Add lhcbdirac

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready for review.

Windows fails but it’s noarch so it doesn’t matter and I’ll probably migrate the setup.py upstream at some point.

(This is the last one in this set!)

File: recipes/lhcbdirac-config/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - ca-policy-lcg
+
+test:
+  commands:
+    - test -f "${CONDA_PREFIX}/etc/dirac.cfg"
+    - echo "${X509_VOMS_DIR}"
+
+about:
+  home: https://lhcb-dirac.readthedocs.io/
+  license: None
```
Comment (ocefpaf) [inline recipes/lhcbdirac-config/meta.yaml]: If there is no license then copyright is automatic. Can you ask open this upstream?

File: recipes/lhcbdirac/setup.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+scripts = glob.glob('%s/LHCbDIRAC/*/scripts/*.py' % base_dir)
+print(scripts)
+
+setup(
+    name='LHCbDIRAC',
+    version='9.3.4',
+    packages=all_packages,
+    scripts=scripts,
+    url='https://lhcb-dirac.readthedocs.io/',
+    license='GPL-3.0',
```
Comment (ocefpaf) [inline recipes/lhcbdirac/setup.py]: I'm not a lawyer but `lhcbdirac-config` should be GPL-3.0 too then, no?

File: recipes/lhcbdirac-config/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/lhcbdirac-config/meta.yaml]: This package (`lhcbdirac-config`) only contains a configuration file that is included in the feedstock directory. All of the code is in the `lhcbdirac` package which is GPL-3.0. I would argue there isn't anything to copyright as it is just contains 5 URLs:

```
DIRAC
{
  Setup = LHCb-Production
  Configuration
  {
    Servers = dips://lbvobox100.cern.ch:9135/Configuration/Server
    Servers += dips://lbvobox101.cern.ch:9135/Configuration/Server
    Servers += dips://lbvobox102.cern.ch:9135/Configuration/Server
    Servers += dips://lbvobox103.cern.ch:9135/Configuration/Server
  }
}
LocalInstallation
{
  Setup = LHCb-Production
  ConfigurationServer = dips://lhcb-conf-dirac.cern.ch:9135/Configuration/Server
}
```
The reason for a separate package is incase this ever changes, but that is unlikely. If you think is need to have a licence attached I'll ask the maintainers of this installation.

Comment (scopatz) [reply recipes/lhcbdirac-config/meta.yaml]: The license is pretty clearly GPL v3, from what I can tell.

Comment (chrisburr) [reply recipes/lhcbdirac-config/meta.yaml]: There are [two recipes here](https://github.com/chrisburr/staged-recipes/tree/add-lhcbdirac/recipes):
* `lhcbdirac`: Clearly GPL-3.0 and the `meta.yaml` reflects this [here](https://github.com/conda-forge/staged-recipes/pull/8387/files#diff-04b7be3e94abf03f8ac73e49a4ea53c6R35)
* `lhcbdirac-config`: Contains only configuration files and currently marked with `license: None`

Comment (ocefpaf) [reply recipes/lhcbdirac-config/meta.yaml]: Indeed but `lhcbdirac-config` is used with `lhcbdirac` and, based on my limited knowledge of GPL-3.0,  `lhcbdirac-config` must be GPL'ed too.

Comment (chrisburr) [reply recipes/lhcbdirac-config/meta.yaml]: I've spoken to the authors and they're happy for it to be GPL 3.

**Comment by scopatz** [comment]
Thanks @chrisburr!

**Pull request merged**